package.json

```diff
@@ -5,11 +5,13 @@
   "license": "GPLv3",
   "description": "Decentralized Messenger",
   "private": true,
+  "type": "module",
   "scripts": {
     "serve": "vite preview",
     "dev": "vite",
     "serve:test": "cross-env ADM_CONFIG_FILE=test vue-cli-service serve",
-    "build": "vue-tsc && vite build",
+    "update-wallets": "node scripts/wallets.js",
+    "build": "npm run update-wallets && vue-tsc && vite build",
```

Comment:
Since the files generated by `scripts/wallets.js` are tracked by git, we don't need to run `update-wallets` in build process. We will `update-wallets` manually and then commit changes.

Reply:
This will also fix the CI/CD build process fail